src/proc_image_processing/cpu/filters/custom/deep_2019_filter.h

```diff
@@ -1,13 +1,14 @@
 /// \author	Antoine Dozois <dozois.a@gmail.com>
 
+// FACTORY_GENERATOR_CLASS_NAME=Deep2019Filter
 
-// FACTORY_GENERATOR_CLASS_NAME=Deep2019
+// FACTORY_GENERATOR_CLASS_NAME=Deep2019Filter
```

Comment:
Why do we need a duplicate of this comment ?

Reply:
mmmh? There is no duplicate, i've removed it!

Reply:
I guess its just a bug from github then..
![image](https://user-images.githubusercontent.com/43056974/126251199-101ac544-5160-4ef3-9ce3-b0a8444a3732.png)